internal/pkg/runtime/engines/singularity/rpc/server/server.go

```diff
@@ -127,8 +127,10 @@ func (t *Methods) LoopDevice(arguments *args.LoopArgs, reply *int) error {
 
 	runtime.LockOSThread()
 	syscall.Setfsuid(0)
+	syscall.Setfsgid(0)
```

Comment:
Should we be setting to `disk` group here? Or change when a new device is created with `Mknod()`?